app/commands/git/sync_track.rb

```diff
@@ -70,8 +70,8 @@ def concept_exercises
 
           # TODO: Remove the || ... once we have configlet checking things properly.
           title: exercise_config[:name].presence || exercise_config[:slug].titleize,
-          taught_concepts: find_concepts(exercise_config[:concepts]),
-          prerequisites: find_concepts(exercise_config[:prerequisites]),
+          taught_concepts: find_concepts(exercise_config[:concepts].to_a),
+          prerequisites: find_concepts(exercise_config[:prerequisites].to_a),
```

Comment:
This is what's causing the error: having a `prerequisites` or `concepts` property returning `nil`

Reply:
I'd suggest moving the `to_a` into `find_concepts` itself, to give a little more protection against bad data being passed in, rather than having to remember to do this everywhere we call the method.